### ISSUE TITLE
Adding logos

### ISSUE THREAD

**Issue by sanjitcodes** [ACM-IGDTUW/Open-Source-Challenge#102]
Adding of logos of ACM(marked with blue) and university(here IGDTUW)(marked with green)

![Screenshot 2020-04-17 at 5 37 14 PM](https://user-images.githubusercontent.com/59444668/79567651-3b06a480-80d2-11ea-96e6-68806cceefd3.png)

**Comment by lisa761** [comment]
Great work! Its good that you have come up with mocks for the UI. I agree with the acm logo and its position but I think we should move the position for IGDTUW logo as it kinda seems abrupt? @soumyaa1804 What do you think?

**Comment by lisa761** [comment]
Also, please edit your description to add `For #93` so that it can create a mention in the Task-5 issue.

**Comment by soumyaa1804** [comment]
Yes, it's a good idea. We have a logo which includes both ACM as well as IGDTUW in it we can use that so that we do not require to add two logos.
